eth/contracts/facets/DFLobbyFacet.sol

```diff
@@ -2,42 +2,50 @@
 pragma solidity ^0.8.0;
 
 // Contract imports
-import {Diamond} from "../vendor/Diamond.sol";
+import {DFDiamond} from "../DFDiamond.sol";
 
 // Interface imports
-import {IDiamondCut} from "../vendor/interfaces/IDiamondCut.sol";
-import {IDiamondLoupe} from "../vendor/interfaces/IDiamondLoupe.sol";
-import {IERC173} from "../vendor/interfaces/IERC173.sol";
+import {IERC173} from "@solidstate/contracts/access/IERC173.sol";
+import {IDiamondReadable} from "@solidstate/contracts/proxy/diamond/readable/IDiamondReadable.sol";
+import {IDiamondWritable} from "@solidstate/contracts/proxy/diamond/writable/IDiamondWritable.sol";
 
 // Storage imports
 import {WithStorage} from "../libraries/LibStorage.sol";
 
 contract DFLobbyFacet is WithStorage {
     event LobbyCreated(address ownerAddress, address lobbyAddress);
 
+    /**
+     * @notice create new diamond with facets from parent diamond
+     *  and optionally execute arbitrary initialization function
+     * @param initAddress optional target of initialization delegatecall
+     * @param initData optional initialization function call data
+     */
     function createLobby(address initAddress, bytes calldata initData) public {
         address diamondAddress = gs().diamondAddress;
-        address diamondCutAddress = IDiamondLoupe(diamondAddress).facetAddress(
-            IDiamondCut.diamondCut.selector
+        address diamondCutAddress = IDiamondReadable(diamondAddress).facetAddress(
+            IDiamondWritable.diamondCut.selector
         );
```

Comment:
I'm thinking about this code and I actually think it needs to be expanded because the new diamond provides additional functionality. The old diamond **only** provided diamondCut but the new one has ownable and stuff